xdsl/ir.py

```diff
@@ -1002,13 +1193,15 @@ def add_ops(self, ops: Iterable[Operation]) -> None:
 
     def insert_ops_before(self, ops: Sequence[Operation],
                           existing_op: Operation) -> None:
-        index = self.get_operation_index(existing_op)
-        self.insert_op(ops, index)
+        for op in ops:
+            self.insert_op_before(op, existing_op)
 
     def insert_ops_after(self, ops: Sequence[Operation],
                          existing_op: Operation) -> None:
-        index = self.get_operation_index(existing_op)
-        self.insert_op(list(ops), index + 1)
+        for op in ops:
+            self.insert_op_after(op, existing_op)
+
+            existing_op = op
 
     def insert_op(self,
```

Comment:
I guess this function is going to be removed as well?

Reply:
yep, defo